src/test/scala/com/databricks/sparkdl/ModelFetcherSuite.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017 Databricks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.databricks.sparkdl
+
+import java.io.{File, FileOutputStream, FileWriter, OutputStream}
+import java.nio.file.{Files, Paths}
+
+import org.apache.commons.io.FileUtils
+import org.scalatest.FunSuite
+
+class ModelFetcherSuite extends FunSuite {
+  val testGraphDefFile = getClass.getResource("/sparkdl/test_net.pb").getFile
+  val testGraphHash = "jVCEKp1bV53eib8d8OKreTH4fHu/Ji5NHMOsgdVwbMg="
+
+  test("Import graph loads a graph only if the hash matches.") {
+    assert(ModelFetcher.importGraph(Paths.get(testGraphDefFile), testGraphHash).isDefined)
+    assert(ModelFetcher.importGraph(Paths.get(testGraphDefFile), "badHash").isEmpty)
+  }
+
+  test("Test getFromWeb using a local url.") {
+    val cacheDir = ModelFetcher.getOrCreateModelCacheDir
```

Comment:
would it be bad to have this cleaned up since it'll force a re-download of potentially large model files? (at least it seems annoying to developers = us) could we use a slightly different directory for testing?

Reply:
👍 Done!